learning_observer/learning_observer/auth/social_sso.py

```diff
@@ -130,6 +154,11 @@ async def social_handler(request):
 
     user = await _google(request)
 
+    # Check roster data source to obtain relevant authorization
+    roster_source = settings.pmss_settings.source(types=['roster_data'])
```

Comment:
Move this line of code to a function and pass the request parameter. For now, the `request` will remain unused, but in the future we will build out this function to properly pull out the correct roster source. Make sure to add a doc string to the function.

Reply:
I added a `handle_lms_request_authorization` function

Reply:
Where is it? Why is not being called here?